packages/fluentui/docs/src/components/ComponentDoc/PrototypePerfExample/PrototypePerfExample.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import ComponentExampleTitle from '../ComponentExample/ComponentExampleTitle';
+
+import { Accordion, Flex, Segment, Menu, Loader } from '@fluentui/react-northstar';
+
+import PrototypeExampleControls from './PrototypePerfControls';
+
+const ComponentPerfChart = React.lazy(async () => ({
+  default: (await import(/* webpackChunkName: "component-chart" */ '../ComponentPerfExample/ComponentPerfChart'))
+    .ComponentPerfChart,
+}));
+const ComponentResourcesChart = React.lazy(async () => ({
+  default: (await import(/* webpackChunkName: "component-chart" */ '../ComponentPerfExample/ComponentResourcesChart'))
+    .ComponentResourcesChart,
+}));
+
+export interface PrototypePerfExampleProps {
+  title: React.ReactNode;
+  description?: React.ReactNode;
+  examplePath: string;
+}
+
+const PrototypePerfExample: React.FC<PrototypePerfExampleProps> = props => {
+  const { title, description, examplePath } = props;
+
+  const Prototype = React.lazy(async () => ({
+    default: (await import(/* webpackChunkName: "prototype-example" */ `../../../examples/${examplePath}`)).default,
```

Comment:
This is what is causing Webpack OOM.

We pass `../../../examples/` as static part of the path.
And `components/PerformanceTests/Performance/Roster.perf` as a variable.
Webpack explodes because of too many possible modules: https://github.com/webpack/webpack/issues/4292#issuecomment-451034292

As a workaround, let's change the split. Edit this line to:
```suggestion
    default: (await import(/* webpackChunkName: "prototype-example" */ `../../../examples/components/PerformanceTests/${examplePath}`)).default,
```

And `examplePath` in packages/fluentui/docs/src/examples/components/PerformanceTests/index.tsx to just `Performance/Roster.perf`